spec/helpers/application_helper/toolbar_chooser_spec.rb

```diff
@@ -54,14 +54,16 @@ def method_missing(sym, *args)
       end
 
       it "should return namespace toolbar on namespace node" do
-        n1 = FactoryGirl.create(:miq_ae_namespace, :name => 'ns1', :parent_id => 1)
+        d1 = FactoryGirl.create(:miq_ae_domain)
+        n1 = FactoryGirl.create(:miq_ae_namespace, :name => 'ns1', :parent_id => d1.id)
```

Comment:
You can also just one-line it and avoid the extra var with:

```ruby
n1 = FactoryGirl.create(:miq_ae_namespace, :name => 'ns1', :parent => FactoryGirl.create(:miq_ae_domain))
```

It's up to you...doesn't matter either way.

Also note that if you move name sequencing into the factory as I has mentioned in the other PR, then you don't need the name here at all.